Packs/Incapsula/Scripts/IncapWhitelistCompliance/IncapWhitelistCompliance.py

```diff
@@ -0,0 +1,106 @@
+import re
+
+from CommonServerPython import *  # noqa: F401
+
+import demistomock as demisto  # noqa: F401
+
+# Consts
+COMPLIANCE_LABEL = "Compliance"
+CURL_GET_COMMAND = 'curl --insecure -L -I -m 1 http://{0} | grep -C 1 "200 OK"'
+HTTP_OK = "200 OK"
+MAIL_TO_RECIPIENT = "IncapsulaUpdates@demisto.com"
+INCAPSULA_WARNING = "****Incapsula Warning****"
+BASIC_WARNING_MAIL = '''
+WARNING
+Your Site {0} is not compliant to the Incapsula allow list policy
+please update your settings
+
+--DBot
+'''
+ADVANCE_WARNING_MAIL = '''
+WARNING
+Your Site {0} is not compliant to the Incapsula allow list policy
+please update your settings as soon as possible!
+
+--DBot
+'''
+
+
+def sendMail(to, subject, body="", bcc=""):
+    return demisto.executeCommand("SendEmail", {'to': to, 'subject': subject, 'body': body, 'bcc': bcc})
+
+
+def escalation(url, severity, owner_mail):
+    if isinstance(owner_mail, list):
+        owner_mail = ''.join([str(x) + ', ' for x in owner_mail[:-1]] + [str(owner_mail[-1])])
+
+    if severity == 0:
+        return None
+    elif severity < 4:
```

Comment:
What's the difference between the block of the `elif` and the `else`?

Reply:
it seems the same removed the elif

Reply:
No, it's a bug (in the original code...)

The elif (meaning 4 or larger) should use the `ADVANCE_WARNING_MAIL` value.